File: C++/program11.cpp

/*
    start
        Call display function
    stop
*/

#include<iostream>
using namespace std;


//Iteration

void Display(int iNo)
{
    int iCnt = 0;

    for(iCnt = 0; iCnt < iNo; iCnt++)
    {
        cout<<"Hello world"<<'\n';
    }
  
}

int main()
{
    int iValue = 0;

    cout<<"Enter frequency"<<'\n';
    cin>>iValue;

    Display(iValue);
    return 0;
}  

File: Java/program11.java

class DisplayClass
{
    public void Display()
    {
        int iCnt = 0;
        for(iCnt = 1; iCnt <= 4; iCnt++)
        {
              System.out.println("Hello");  
        }
     


    }
}

class program11
{
    public static void main(String A[])
    {
        DisplayClass dobj = new DisplayClass();
        dobj.Display();

        
    }
}